.github/workflows/docs-preview-trigger.yml

```diff
@@ -4,11 +4,13 @@ on:
     pull_request:
         paths:
             - 'docs/published/**'
+            - 'docs/onboarding/**'
```

Comment:
I wonder if just `docs/**` is enough here, it's ok if we build it slightly too often when we change inside `/internal`

Reply:
🤔 I'm not against it, any reason not to be more specific tho? I think internal is not meant to trigger builds. tagging @webjunkie for more context here